lib/mock-nexmo.js

```diff
@@ -37,8 +52,16 @@ function MockNexmo(log, balanceThreshold) {
 
         log.info({ op: 'sms.send.mock' })
 
-        callback(null, {
-          messages: [{ status: '0' }]
+        // HACK: Enable remote tests to see what was sent
```

Comment:
The reason I consider it a hack is because it meant passing the full config into the SMS module, wherease before it only had access to `config.sms`. Ideally we'd have a restsms service to do an equivalent job for us, but that seems unlikely for now.